web/src/js/ducks/flows.ts

```diff
@@ -23,13 +24,15 @@ export interface FlowsState extends store.State<Flow> {
     filter?: string;
     sort: { column?: keyof typeof sortFunctions; desc: boolean };
     selected: string[];
```

Comment:
I'm confused why here `selected` has always been a `string[]` but in a different part `select` was `Flow` and now became `selectedFlows` as `string[]`?